src/test/java/rest/gorestCoIn/APIRest.java

```diff
@@ -0,0 +1,60 @@
+package rest.gorestCoIn;
+
+import io.restassured.RestAssured;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import rest.gorestCoIn.utils.User;
+import rest.gorestCoIn.utils.helperMethodsForGorestCoIn.APICal;
+
+public class APIRest{
+
+    @BeforeClass
+    public void setup() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+
+    }
+
+    @Test
+    public void postUser() {
+
+        Header header = new Header("Authorization", "Bearer 73e824b7f681b6e4d47c97ed07c2e26f6f91effd9a3496aad4f4960798709544");
+
+        String body = "{\n" +
+                      "\"name\": \"tester\",\n" +
+                      "\"email\": \"tesdsdsdsstssststs@test.io\",\n" +
```

Comment:
Though your test deletes the created user, there is always a risk of test failure, which will bring to data issue in your test later. Also, there is a risk, that the same e-mail has been already used by someone else.
In such cases, there is a good practice to append a timestamp to the username. You can get the current date-time and append it and get a unique name always.